.github/pull_request_template.md

```diff
@@ -6,8 +6,8 @@ Describe Change Here!
 
 # Implementation
 
-* WPT tests: [LINK]()
+* WPT tests:
```

Comment:
- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/10
- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/11
- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/12
- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/14
- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/15
- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/16
- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/17

And maybe:
- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/19
- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/20